pkg/controller/core.oam.dev/oamruntime_controller.go

```diff
@@ -91,3 +93,23 @@ type Args struct {
 	// IgnoreDefinitionWithoutControllerRequirement indicates that trait/component/workflowstep definition controller will not process the definition without 'definition.oam.dev/controller-version-require' annotation.
 	IgnoreDefinitionWithoutControllerRequirement bool
 }
```

Comment:
oamruntime_controller is not in maintain now.

Reply:
@Somefive So there is no need to do about the initialization of args?

Reply:
Yes. Core controller and APIServer are good enough. Other legacy entrances are expected to be deprecated.

Reply:
OK, but as far as I know, these parameters are needed when starting application_controller